### ISSUE TITLE
Is version 2 still beta?

### ISSUE THREAD

**Issue by RobSmyth** [gsscoder/commandline#496]
Hi, 

Is there a stable v2 release?

It looks to me that the last stable release was 1.9.71 (2013?) and 2.1.1-beta was published on NuGet Nov last year.

Thanks

**Comment by nemec** [comment]
No, there is not a stable 2.x release, but the beta is pretty "stable" from a random-crashing perspective (and it contains a lot of bug fixes that the 1.x doesn't)

**Comment by jmecosta** [comment]
@nemec any estimates when 2.x is release. would like to take that into use

**Comment by ohadschn** [comment]
See https://github.com/gsscoder/commandline/issues/488
